python/freetensor/core/optimize.py

```diff
@@ -48,6 +49,9 @@ def f(...):
         The target architecture. You don't have to set target if you set device
     device : Device (Optional)
         Where to run the program
+    default_dynamic_range : bool
+        If True, the built-in range is replaced with freetensor.dynamic_range.
+        Defaults to True
```

Comment:
How about adding a `freetensor.static_range`, which is simply a reference to `builtins.range`? Then users may explicitly use `freetensor.static_range` or `freetensor.dynamic_range`, or they can always use `range` but control the behavior via `default_dynamic_range`.

Reply:
Will add it.